src/ariel-os-embassy-common/src/identity.rs

```diff
@@ -4,6 +4,38 @@
 //! of this API.
 #![deny(missing_docs)]
 
+/// A EUI-48 identifier, commonly known as a MAC address.
+///
+/// The allowed value space of this type is identical to its inner type, but the use of an address
+/// in this types implies awareness of its [bits'
+/// semantics](https://en.wikipedia.org/wiki/MAC_address#Address_details).
+// This type may be moved into a more common place in Ariel if it gets used more widely; until
+// then, this is the place where it is needed and implemented.
+#[derive(Copy, Clone, PartialEq)]
+pub struct Eui48(pub [u8; 6]);
```

Comment:
I did consider this, but given that it's very hard to imagine any other representation, I think that's fine.

If we went with a closed type, we'd should also have a from and into, and at that point it gets very verbose for what can otherwise be a single line.